src/host/srvinit.cpp

```diff
@@ -439,11 +441,7 @@ try
 
     if (!g.handoffTerminalClsid)
     {
-        TraceLoggingWrite(g_hConhostV2EventTraceProvider,
-                          "SrvInit_ReceiveHandoff_NoTerminal",
-                          TraceLoggingLevel(WINEVENT_LEVEL_VERBOSE),
-                          TraceLoggingKeyword(TIL_KEYWORD_TRACE));
-        return E_NOT_SET;
+        g.handoffTerminalClsid = CLSID_SystemDelegationTerminal;

```

Comment:
I think we should introduce something like a `IConsoleHandoff2` interface as a follow-up that provides the correct `handoffTerminalClsid` directly as a handoff-parameter.
It might make sense to wait a little bit for that so that we can see whether any other parameters should be added...